inception-ui-external-search/NOTICE.txt

```diff
@@ -0,0 +1,9 @@
+== Flatastic Icon Set 5
+
+http://www.customicondesign.com/free-icons/flatastic-icon-set/flatastic-part-5/
```

Comment:
Please use only icons which are free for any purpose. This icon set requires buying a license for commercial use. Best a non-restrictive Creative Commons like CC-0 or CC-BY.

Based on your choice of the icon motive, I would suggest the 32x32 `world_go.png` icon from https://www.fatcow.com/free-icons

When the icon is replaced and the NOTICE.txt file, this PR will be approved.

Reply:
Updated to the `world_go.png` icon